core/src/com/unciv/logic/GameStarter.kt

```diff
@@ -173,6 +177,25 @@ object GameStarter {
         }
     }
 
+    private fun addCivPolicies(gameInfo: GameInfo, ruleset: Ruleset) {
+        for (civInfo in gameInfo.civilizations.filter { !it.isBarbarian() }) {
+
+            // generic start with policy unique
+            for (unique in civInfo.getMatchingUniques(UniqueType.StartsWithPolicy)) {
+                // get the parameter from the unique
+                val policyName = unique.params[0]
+
+                // check if the policy is in the ruleset and not already adopted
+                if (ruleset.policies.containsKey(policyName) && !civInfo.policies.isAdopted(policyName)) {
+                    civInfo.policies.freePolicies++
+                    val policyToAdopt = ruleset.policies[policyName]
+                    if (policyToAdopt != null)
```

Comment:
This if will always be false, stove we already checked it exists in the previous if

Reply:
Always true, sorry
That is to say we can remove the check

Reply:
At that point kotlin _will_ infer policyToAdopt as Policy? and thus insist on something...

Reply:
I was trying to avoid nullable type mismatch back then when I wrote that... Yeah perhaps I'll just change it to something simpler, such as `val policyToAdopt = ruleset.policies[policyName]!!`.